ui/src/app/cohort-search/charts/charts.component.ts

```diff
@@ -0,0 +1,77 @@
+import {Component, ChangeDetectionStrategy, Input} from '@angular/core';
+import {Set} from 'immutable';
+
+
+@Component({
+  selector: 'app-charts',
+  template: `
+    <div class="box">
+      <app-gender-chart [data]="genderCounts">
+      </app-gender-chart>
+    </div>
+    <div>&nbsp;</div>
+    <div class="box">
+      <app-race-chart [data]="raceCounts">
+      </app-race-chart>
+    </div>
+  `,
+  styles: [`.box { border: 1px solid #d7d7d7; }`],
```

Comment:
I tend to prefer having html and css files instead of inlining them. It can get messy as the app grows and we have to keep changing the html and adding styles. In this case it may be fine since charts probably won't change that much.